plugins/woocommerce/changelog/fix-order_key_migration

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Remove unique constraint from order_key, since orders can be created with emtpy order key, which conflicts with the constraint.
```

Comment:
Nit:
```suggestion
Remove unique constraint from order_key, since orders can be created with empty order keys, which then conflict with the constraint.
```